convert2rhel/utils.py

```diff
@@ -560,4 +560,17 @@ def backup(self):
             loggerinst.warning("Can't access %s" % TMP_DIR)
 
 
+def check_if_mount_is_readonly():
+    loggerinst = logging.getLogger(__name__)
+
+    mounts = get_file_content('/proc/mounts', as_list=True)
+    for line in mounts:
+        device, mount_point, filesystem, flags, __, __ = line.split()
+        flags = flags.split(',')
+        if mount_point == '/mnt' or mount_point == '/sys':
+            (loggerinst.critical(
+                "%s are not read-write. Conversion can not continue." % mount_point) if 'ro' in flags else loggerinst.debug(
+                "/mnt and /sys mount are readwrite."))
```

Comment:
As you have it now, the _"/mnt and /sys mount are readwrite."_ would be printed twice.

```suggestion
        if 'ro' not in flags and mount_point not in ('/mnt',  '/sys'):
            continue
        if mount_point == '/mnt':
            loggerinst.critical("/mnt/ is read-only - that would cause the conversion to fail.\n"
                                "Do not mount directly to /mnt,  mount to its subdirectories instead.")
        else:  # /sys
            loggerinst.critical("/sys/ is read-only - that would cause the conversion to fail.\n"
                                "Make sure it is mounted in read-write mode before executing convert2rhel.")
        loggerinst.debug("%s mounted in read-write mode." % mount_point)
```